presto-main/src/main/java/com/facebook/presto/execution/buffer/LazyOutputBuffer.java

```diff
@@ -190,6 +202,12 @@ public void setOutputBuffers(OutputBuffers newOutputBuffers)
         }
     }
 
+    private SpoolingOutputBuffer createSpoolingOutputBuffer(OutputBuffers outputBuffers)
+    {
+        TempStorage tempStorage = new LocalTempStorage(ImmutableList.of(createTempDir().toPath()), 1);
```

Comment:
Let's make the `1` argument a configuration property.  It should probably be defaulted to 0.95 or 0.99.